### ISSUE TITLE
Possibility to define output format

### ISSUE THREAD

**Issue by gregzawadzki** [skiptirengu/media-split#1]
Hello @skiptirengu and thank you for this script.

FFMPEG is powerfull tool. Maybe a way for us to define output file format?
I can convert m4a into mp3... but would be fun if it could stay m4a :) 👍

**Comment by sugiruu** [comment]
Sure thing, I'll look into it when I have some free time.

**Comment by sugiruu** [comment]
Fixed on v4.1.0. You should now be able to specify the output format using the -f parameter.
```bash
mp3-split -i "/home/music/input.3gpp" -o "/home/music/folder" -f m4a
```

**Comment by gregzawadzki** [comment]
nice, fast ;) Thank you! 🥇
